### ISSUE TITLE
CPM 1.20.4 The loading screen does not disappear

### ISSUE THREAD

**Issue by kirillka77734** [tom5454/CustomPlayerModels#642]
After: ‘It took *** seconds to start the game’ the bar refuses to go further and the menu with the choice of network game or single game does not appear further. The loading screen does not disappear. If you remove the CPM from the mod build, the game will run smoothly and everything will be fine (regardless of whether you even installed Pehkui or not). I searched for a solution to the problem on github and discord, but never found one(or didn't look hard enough).
https://mclo.gs/FcgElGa
![image](https://github.com/user-attachments/assets/9ea4632d-e662-4523-8d08-33e8185ddff2)

after ‘[22:52:22] [Render thread/WARN]: Game took 66.929 seconds to start’ the bar hung in place (as on the screenshot), it didn't move anymore. After 30 seconds I closed the game.

I tried to wait for the strip to come to the end again, but it never did. I waited after ‘’It took *** seconds to start the game’ and for 10 minutes, even more, but it didn't help.

Even when I removed the custom resourcepacks, and that didn't help
logs without custom resource packs: https://mclo.gs/6qt3HjM

**Comment by tom5454** [comment]
Remove `cosmetica`
Duplicate of #261
